tests/bcl-test/BCLTests/templates/common/TestRunner.NUnit/NUnitTestRunner.cs

```diff
@@ -169,7 +169,6 @@ public void TestFinished (ITestResult result)
 					sb.Append ("\t[INFO] ");
 					break;
 				}
-				TotalTests++;
```

Comment:
Issue found will testing the feature, we were reporting two times the real number of tests.